src/api/extendOptions.ts

```diff
@@ -83,5 +83,12 @@ export default async (opts?: SupiOptions): Promise<StrictSupiOptions> => {
     extendedOpts.prefix = path.join(extendedOpts.prefix, subfolder)
   }
   extendedOpts.rawNpmConfig['registry'] = extendedOpts.registry
+  if (!extendedOpts.rawNpmConfig['unsafe-perm']) {
+    extendedOpts.rawNpmConfig['unsafe-perm'] = process.platform === 'win32' ||
```

Comment:
I added `unsafePerm` to the options https://github.com/pnpm/types/commit/a35d2a3376b9ec211b4ffb9a602237d67eeea6ef

So just update `@pnpn/types` and use it as a regular option.